operators/paint.py

```diff
@@ -50,8 +50,13 @@ def execute(self, context):
 
         # create or get image
         if not tex.image:
-            img = utils_paint.create_image("image", self.resolution, self.color, channel.is_data)
-            tex.image = img
+            # Blanchsb code fix Dec2021: Painting masks throws error #17
```

Comment:
You could remove these comments, other than that it looks good. Also the pull request wasn't linked to the issue, if you want to have another go at doing that go ahead, otherwise I just linked the issue to the pull request.